.drone.yml

```diff
@@ -50,18 +50,6 @@ steps:
       TAGS: bindata sqlite sqlite_unlock_notify
     depends_on: [deps-backend]
 
-  - name: lint-backend-windows
-    image: gitea/test_env:linux-amd64  # https://gitea.com/gitea/test-env
-    commands:
-      - make golangci-lint vet
-    environment:
-      GOPROXY: https://goproxy.cn # proxy.golang.org is blocked in China, this proxy is not
-      GOSUMDB: sum.golang.org
-      TAGS: bindata sqlite sqlite_unlock_notify
-      GOOS: windows
```

Comment:
Why no more windows checks?

Reply:
Nvm, saw your comment.

Reply:
Tried something but didn't work. I think this is an unsolvable issue as one can not say to build the tool for one OS, but lint for another. I always felt it wrong that `GOOS` even has influence on the linting (I'm not sure it actually does). Essentially this command does not work on non-Windows:

```
GOOS=windows go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.0 --version
```

Reply:
Tried something but didn't work. I think this is an unsolvable issue as one can not say to build the tool for one OS, but lint for another. I always felt it wrong that `GOOS` even has influence on the linting (I'm not sure it actually does). Essentially this command does not work on non-Windows:

```
GOOS=windows go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.0 --version
```

Reply:
`go help run` indicates something that almost sounds it might work, but I fail to understand how one is supposed to use it:

```
By default, 'go run' runs the compiled binary directly: 'a.out arguments...'.
If the -exec flag is given, 'go run' invokes the binary using xprog:
	'xprog a.out arguments...'.
If the -exec flag is not given, GOOS or GOARCH is different from the system
default, and a program named go_$GOOS_$GOARCH_exec can be found
on the current search path, 'go run' invokes the binary using that program,
for example 'go_js_wasm_exec a.out arguments...'. This allows execution of
cross-compiled programs when a simulator or other execution method is
available.
```